java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -144,6 +150,63 @@ public T set(String key, String value) {
         return getThis();
     }
 
+    /**
+     * Retrieve the value associated with <code>field</code> in the hash stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/hget/">redis.io</a> for details.
+     * @param key The key of the hash.
+     * @param field The field in the hash stored at <code>key</code> to retrieve from the database.
+     * @return The value associated with <code>field</code>, or <code>null</code> when <code>field
```

Comment:
Please change `return` to `Commands Response` to be aligned [python](https://github.com/aws/glide-for-redis/blob/7b4c2f09e30ae399f63e248baebaaca1acd3fcd7/python/python/glide/async_commands/transaction.py#L308) and [Node](https://github.com/aws/glide-for-redis/blob/7b4c2f09e30ae399f63e248baebaaca1acd3fcd7/node/src/Transaction.ts#L310) wrappers.